framework/include/utils/SlepcSupport.h

```diff
@@ -15,7 +15,7 @@
 #ifndef SLEPCSUPPORT_H
```

Comment:
this commit should not exist and changes here should be amendment in the previous commits.